Add `COEGTS 2025: Registrations Now Open!` news post (#13439)
ppy/osu-wiki: news/2025/2025-06-29-cavoes-osu-events-global-taiko-showdown-2025-registrations-now-open.md

---
layout: post
title: "cavoe's osu! event's Global Taiko Showdown 2025: Registrations Now Open!"
date: 2025-06-29 21:00:00 +0000
---

We invite you to witness the final tournament of GTS, and IRL at that, with the second edition of the cavoe's osu! event's Global Taiko Showdown 2025 happening in the Netherlands!

![](/wiki/shared/news/2025-06-29-cavoes-osu-events-global-taiko-showdown-2025-registrations-now-open/banner.png)

The [cavoe's osu! event's Global Taiko Showdown](https://gratiaproductions.com/coegts/home) has been a tournament that showcased its strengths into providing some of the most interesting tournament action in osu!taiko back in 2023, while being its first ever LAN tournament as well, and being the only tournament in the game with, truly, **100% original content.**

So, there is no better way to end GTS than with this tournament.

> Regarding the tournament's bracket format, there is a slight change from the past year. This is planned to be a **single-elimination bracket** of **8 players** that will be in the tournament following the online Qualifiers, which will be streamed, due to the nature of the songs in this mappool.

Reviewer: ```suggestion
Regarding the tournament's bracket format, there is a slight change from the past year. This is planned to be a **single-elimination bracket** of **8 players** that will be in the tournament following the online Qualifiers. Due to the nature of the songs in this mappool (they're fully original creations), matches will be livestreamed right from the get-go.
```

> As a special event as well this year, the **Qualifier Results** will be showcased at COE itself, along with the **Main Mappool Showcase.** 

Reviewer: we wouldn't consider these proper nouns

```suggestion
As a special event as well this year, the **Qualifier results** will be showcased at COE itself, along with the **main mappool showcase**.
```

The tournament's detailed schedule and all relevant info for participating can be found below.

## Tournament schedule

> | Event | Timestamp |
> | --: | :-- |
> | Registration phase | June 29 (21:00 UTC) – July 13 (23:59 UTC) |
> | Qualifier showcase | July 13 (15:00 UTC) |
> | Qualifier stage | July 14 to June 20 |
> | Qualifier Results & Main Mappool showcase | July 28 (17:00 UTC) |
> | Day 1 (Quarterfinals) | July 31 (15:00-20:00 UTC) |
> | Day 2 (Semifinals) | August 1 (17:30-20:30 UTC) |
> | Day 3 (Match for 3rd place & Grand Finals) | August 2 (8:30-10:00 UTC & 13:30-15:30 UTC) |

Reviewer: ```suggestion
| Event | Timestamp (UTC) |
| --: | :-- |
| Registration phase | June 29 (21:00) – July 13 (23:59) |
| Qualifier showcase | July 13 (15:00) |
| Qualifier stage | July 14 to June 20 |
| Qualifier results & main mappool showcase | July 28 (17:00) |
| Day 1 (Quarterfinals) | July 31 (15:00–20:00) |
| Day 2 (Semifinals) | August 1 (17:30–20:30) |
| Day 3 (Match for 3rd place & Grand Finals) | August 2 (8:30–10:00 & 13:30–15:30) |
```

## How do I register?

> Registrations are currently open on the [GTS website](https://gratiaproductions.com/coegts/home). Make sure to read all the rules for this tournament on there, have a COE ticket (whether BYOC or Event) handy for July 31st to August 2nd, and join the [GTS Discord server](https://discord.com/invite/3mGC3HB) to join us in this final experience!

Reviewer: ```suggestion
Registrations are currently open on the [GTS website](https://gratiaproductions.com/coegts/home). Make sure to read all the rules for this tournament on there, have a [COE ticket](https://cavoe.events/) (whether BYOC or Event) handy for July 31st to August 2nd, and join the [GTS Discord server](https://discord.com/invite/3mGC3HB) to join us in this final experience!
```

**The deadline for registering to this tournament is July 13, 23:59 UTC.**

As long as you have done everything above, you will be instantly noted on the website as registered!

> ***Registration attempts after this deadline will not be accepted whatsoever. Again, this is the final iteration of COEGTS, so there will only be two weeks to consider participation.***

Reviewer: ```suggestion
***Registration attempts after this deadline will not be accepted whatsoever. Again, this is the final iteration of COEGTS and GTS as a series, so there will only be two weeks to consider participation.***
```
